src/pages/my_registrations/MyRegistrationsList.tsx

```diff
@@ -1,62 +1,28 @@
 import { Typography } from '@mui/material';
-import { useEffect, useState } from 'react';
+import { UseQueryResult } from '@tanstack/react-query';
+import { useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import { useDispatch } from 'react-redux';
 import { deleteRegistration } from '../../api/registrationApi';
+import { ResultParam, ResultSearchOrder } from '../../api/searchApi';
 import { ConfirmDialog } from '../../components/ConfirmDialog';
-import { ListPagination } from '../../components/ListPagination';
-import { RegistrationList } from '../../components/RegistrationList';
+import { SortSelector } from '../../components/SortSelector';
 import { setNotification } from '../../redux/notificationSlice';
-import {
-  emptyRegistration,
-  Registration,
-  RegistrationPreview,
-  RegistrationSearchItem,
-} from '../../types/registration.types';
-import { isErrorStatus, isSuccessStatus, ROWS_PER_PAGE_OPTIONS } from '../../utils/constants';
+import { SearchResponse2 } from '../../types/common.types';
+import { RegistrationAggregations, RegistrationSearchItem } from '../../types/registration.types';
+import { isErrorStatus, isSuccessStatus } from '../../utils/constants';
 import { getIdentifierFromId } from '../../utils/general-helpers';
-import { stringIncludesMathJax, typesetMathJax } from '../../utils/mathJaxHelpers';
-import { convertToRegistrationSearchItem, getTitleString } from '../../utils/registration-helpers';
+import { getTitleString } from '../../utils/registration-helpers';
+import { RegistrationSearch } from '../search/registration_search/RegistrationSearch';
 
 interface MyRegistrationsListProps {
-  registrations: RegistrationPreview[];
-  refetchRegistrations: () => void;
+  registrationsQuery: UseQueryResult<SearchResponse2<RegistrationSearchItem, RegistrationAggregations>>;
```

Comment:
Vurder om man skal importere samme her som i `RegistrationSearch.tsx`. Kanskje burde typen komme fra en `types`-fil for alle plassene i stedet for å avhenge av `SearchPage` som virker som en sirkulær avhengighet 🤔

Reply:
Får ingen klager på det i bygget, så rydder opp litt i gjenbruk av selve responstypen, men ellers tenker jeg da det er greit